src/main/java/com/lambdaschool/oktafoundation/controllers/ApplicationController.java

```diff
@@ -100,11 +99,11 @@
 
         newapp.setAppid(0);
         // could be save or saveByAuth
-        newapp = appService.saveByAuth(u,newapp);
+        newapp = appService.saveByAuth(u, newapp);
 
         HttpHeaders responseHeaders = new HttpHeaders();
         URI newOrgURI = ServletUriComponentsBuilder.fromCurrentRequest()
-            .path("/{appid}")
+            .path("/appid")
```

Comment:
I think this might return the literal string "appid" rather than the actual `appid` But this is just the response header, so if it's broken it's not a big deal.